ui/ui-components/pages/_app.tsx

```diff
@@ -10,21 +10,26 @@ import PageTransition from "../components/PageTransition";
 import StatusSubscription from "../components/StatusSubscription";
 import "../components/Icons";
 
+import { GrouparooNextPageProps, GrouparooWebAppProps } from "../types/app";
+
 import { Actions } from "../utils/apiData";
 import * as eventHandlers from "../utils/eventHandlers";
 import { getModelFromUrlOrCookie, setModelCookie } from "../utils/modelHelper";
 
-export default function GrouparooWebApp(props) {
+export default function GrouparooWebApp(
+  props: AppProps & GrouparooWebAppProps & { err: any }
+) {
   const { Component, pageProps, err, hydrationError } = props;
 
-  const combinedProps = Object.assign({}, pageProps || {}, {
+  const combinedProps: GrouparooNextPageProps<any> = {
```

Comment:
I have to set this as `any` here because when I try to inject the `err` prop when rendering the `<Component`, it causes a typing error. I'm still trying to figure out how these all come together.